tox.ini

```diff
@@ -45,7 +45,7 @@ commands =
 ;    remove specific directory targets for testing once we can
 ;    install and test with aircraft model deps i.e. Detectron2
 ;    pytest --cov-config .coveragerc --cov aws.osml --cov-report term-missing {posargs}
-    pytest test/aws/osml/models/centerpoint test/aws/osml/models/flood --cov-config .coveragerc --cov aws.osml --cov-report term-missing {posargs}
+    pytest test/aws/osml/models/centerpoint test/aws/osml/models/centerpoint_segmentation test/aws/osml/models/flood --cov-config .coveragerc --cov aws.osml --cov-report term-missing {posargs}
```

Comment:
Are we missing the `aircraft` as well? We should test the `aircraft` package.

Reply:
The aircraft model won't test with Tox atm